tldw_Server_API/app/main.py

```
@@ -3843,18 +4065,24 @@ async def _claims_rebuild_loop():
             logger.info("WebSub renewal worker cancelled")
         # Stop usage aggregators gracefully
         try:
-            if "usage_task" in locals() and usage_task:
-                from tldw_Server_API.app.services.usage_aggregator import stop_usage_aggregator as _stop_usage
+            if "usage_aggregator" not in coordinated_legacy_component_names:
+                if "usage_task" in locals() and usage_task:
```

Ruff (0.15.7)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)